pkg/ovsdb/monitor.go

```diff
@@ -562,6 +564,37 @@ func unmarshalData(data []byte) (map[string]interface{}, error) {
 	return obj, nil
 }
 
+func (u *updater) isRowAppearOnWhere(data map[string]interface{}) (bool, error) {
+	if u.mcr.Where == nil {
+		return true, nil
+	}
+	var cond bool
+	for i := 0; i < len(*u.mcr.Where); i++ {
+		switch reflect.TypeOf((*u.mcr.Where)[i]).Kind() {
+		case reflect.Slice:
+			condition, ok := ((*u.mcr.Where)[i]).([]interface{})
+			if !ok {
+				return false, errors.New("unvalid condition")
+			}
+			log := klogr.New() // TODO: propogate real loger insted of this generic one
+			res, err := NewCondition(u.tableSchema, nil, condition, log)
```

Comment:
instead of nil, I would add empty namedUUIDResolver, it can prevent NPE

Reply:
Done